layouts/shortcodes/cloud-siem-aws-setup-cloudformation.en.md

```diff
@@ -1,14 +1,17 @@
 1. Go to Datadog's [AWS integration tile][102] to install the integration.
-2. Click **Automatically Using CloudFormation**. If there is already an AWS account set up, click **Add Another Account** first.
-3. Select the AWS Region where the CloudFormation stack will be launched.
-4. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
-5. Select **Yes** for *Send Logs to Datadog* to set up the Datadog Lambda Forwarder to be used later for sending AWS CloudTrail logs to Datadog.
-6. Click **Launch CloudFormation Template**. This opens the AWS Console and loads the CloudFormation stack with the parameters filled in based on your selections in the Datadog form. 
+1. Click **Automatically Using CloudFormation**. If there is already an AWS account set up, click **Add Another Account** first.
+1. Select the AWS Region where the CloudFormation stack will be launched.
+1. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
+1. To configure the Datadog Lambda Forwarder, select **Yes** for **Send Logs to Datadog**. This enables AWS CloudTrail logs to be sent to Datadog.
+1. To enable Cloud Security Management, select **Yes** for **Detect security issues**.
+1. The following option for Agentless Scanning appears when you select **Yes** for **Detect security issues**.
+  - **Enable Vulnerability Management (Host, Container and Lambda)**: Switch the toggle to the on position to scan for risks and vulnerabilities using Agentless Scanning.
```

Comment:
```suggestion
  - **Enable Vulnerability Management (Host, Container, and Lambda)**: Switch the toggle to the on position to scan for risks and vulnerabilities using Agentless Scanning.
```

Reply:
I agree with this one; however, this is how it's listed in the UI, so I'd like to keep it like this for now.